packages/docusaurus-plugin-sitemap/src/options.ts

```diff
@@ -14,6 +14,12 @@ export type PluginOptions = {
   changefreq: EnumChangefreq;
   /** @see https://www.sitemaps.org/protocol.html#xmlTagDefinitions */
   priority: number;
+  /**
+   * Enables or disables generating lastmod; if set to `date`, lastmod will be
+   * truncated to date and not include the timestamp.
+   * @see https://www.sitemaps.org/protocol.html#xmlTagDefinitions
+   */
+  lastmod: boolean | 'date';
```

Comment:
Wonder if we really need that option? 🤔 

Maybe it could be useful but afaik nobody asked for this yet